src/System.Private.CoreLib/src/System/Diagnostics/Eventing/RuntimeEventSourceHelper.Unix.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+
+namespace System.Diagnostics.Tracing
+{
+    internal sealed class RuntimeEventSourceHelper
+    {
+        internal static Interop.Sys.ProcessCpuInformation cpuInfo = new Interop.Sys.ProcessCpuInformation();
```

Comment:
```suggestion
        internal static Interop.Sys.ProcessCpuInformation cpuInfo;
```

It should be fine to depend on the default zero-initialization here.